src/extension.ts

```diff
@@ -74,30 +75,35 @@ export function activate(context: vscode.ExtensionContext): void {
     context.subscriptions.push(vscode.commands.registerCommand(
         `${SETTINGS_VIEW_NAME}.refresh`,
         () => {
+            telemetryReporter.sendTelemetryEvent('user/buttonPress', { 'VSCode.buttonCode': buttonCode.refreshTargetList });
```

Comment:
As a follow up, I wonder if  we should enumerate these strings in a similar manner to the buttonCode enum. Doesn't need to be in this PR though, as there is alot of pre-existing telemetry that would be affected